### PR TITLE
Update microsoft-defender-endpoint-ios.md

### PR DESCRIPTION
**Pull request by shthota77**

### CODE REVIEW

**Review by shthota77 — COMMENTED** [review]
MAM changes

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md [hunk 1]
```diff
@@ -57,7 +57,8 @@ ms.technology: mde
 - Access to [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization.
 
     > [!NOTE]
-    > Microsoft Intune is the only supported Unified Endpoint Management (UEM) solution for deploying Microsoft Defender for Endpoint and enforcing Defender for Endpoint related device compliance policies in Intune.
+    > Microsoft Defender for Endpoint now extends support to organization’s data within a managed application for those who aren’t using an MDM but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [Mobile Application Management (MAM).](https://docs.microsoft.com/en-us/mem/intune/apps/mam-faq)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md]:
```suggestion
    > Microsoft Defender for Endpoint now extends support to an organization’s data within a managed application for those who aren’t using mobile device management (MDM) but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [mobile application management (MAM)](/mem/intune/apps/mam-faq).
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md [hunk 2]
```diff
@@ -57,7 +57,8 @@ ms.technology: mde
...
+>Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune mobile device management (MDM) only.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md]:
```suggestion
    > Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune MDM only.  
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md [hunk 3]
```diff
@@ -82,3 +83,4 @@ For more information, see [Deploy Microsoft Defender for Endpoint on iOS](ios-in
 - [Configure Microsoft Defender for Endpoint on iOS features](ios-configure-features.md)
 - [Configure app protection policy to include Defender for Endpoint risk signals (MAM)](ios-install-unmanaged.md)
 - [Configure Conditional Access policy based on device risk score from Microsoft Defender for Endpoint](ios-configure-features.md#conditional-access-with-defender-for-endpoint-on-ios)
+- [What is mobile app management in Microsoft Intune? | How to get started with MAM.](https://docs.microsoft.com/en-us/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md]:
```suggestion
- [Mobile Application Management (MAM) basics](/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shthota77 - kindly commit the suggested changes. Many thanks.

**Pull request merged**